updatecli/updatecli.d/major-kubewarden-update-with-crd-update.yaml

```diff
@@ -6,6 +6,8 @@ sources:
     kind: yaml
     transformers:
       - semverinc: '{{ requiredEnv .semverinc }}'
+      - addsuffix: '{{ requiredEnv .prerelease_suffix }}'
+      - trimsuffix: ' '
```

Comment:
@olblak is there a way to say to updatecli to add a suffix **only** if there is something in the environment variable defined in `.prerelease_suffix`?